config/pipelines/high_throughput_lcm_triomics.wip.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Pipeline configuration for EMSeq
+LCM Triomics:
```

Comment:
As in the SS PR, consider adding 'EM-Seq' into the pipeline name - because in Y24-266 further EM-Seq-specific relationships will be added.

Reply:
Oh and as per SS PR, consider adding a library type filter

Reply:
Pipeline (change the key)

LCM Triomics EMSeq

Reply:
Add 

pipeline_group: LCM Triomics WGS and EMSeq